coordinator/internal/logic/provertask/batch_prover_task.go

```diff
@@ -146,13 +144,14 @@ func (bp *BatchProverTask) Assign(ctx *gin.Context, getTaskParameter *coordinato
 
 	if _, ok := taskCtx.HardForkNames[hardForkName]; !ok {
 		bp.recoverActiveAttempts(ctx, batchTask)
-		log.Error("incompatible prover version",
+		log.Debug("incompatible prover version",
```

Comment:
this is useful for debug. could we keep it as info?

Reply:
It will keep spamming the logs, tho. Are you sure you want that?

Reply:
yes, info will be spamming info